pages/profiles/[publicKey]/offers.tsx

```diff
@@ -67,6 +67,11 @@ const OfferPage: NextPage<WalletDependantPageProps> = ({ publicKey, ...props })
 
   const offerCount = receivedCount + sentCount;
 
+  function byDate(a: any, b: any) {
+    //chronologically by year, month, then day
+    return new Date(a.createdAt).valueOf() - new Date(b.createdAt).valueOf(); //timestamps
```

Comment:
Please use .getTime() insetad of .valueOf() -> https://bobbyhadz.com/blog/javascript-convert-iso-date-to-timestamp#:~:text=Use%20the%20getTime()%20method,uses%20UTC%20for%20time%20representation.

or look into using the library https://moment.github.io/luxon/ like we use other places, search for `import { DateTime } from 'luxon'`